pkg/test/environment.go

```diff
@@ -60,19 +60,29 @@ type Environment struct {
 	ImageResolver          *imagefamily.Resolver
 	LaunchTemplateProvider *launchtemplate.Provider
 	LoadBalancerProvider   *loadbalancer.Provider
+
+	// Settings
+	nonZonal bool
 }
 
 func NewEnvironment(ctx context.Context, env *coretest.Environment) *Environment {
+	return NewRegionalEnvironment(ctx, env, fake.Region, false)
+}
+
+func NewEnvironmentNonZonal(ctx context.Context, env *coretest.Environment) *Environment {
+	return NewRegionalEnvironment(ctx, env, "westcentralus", true)
```

Comment:
nit: maybe have a `fake.NonZonalRegion`?

Reply:
Extracted const. (May still need a better location for it.)